packages/x-components/static-docs/experience-search-&-discovery/search-box.md

```diff
@@ -0,0 +1,81 @@
+they're looking for in your shop. It usually includes an input field, a search button, and a clear
+button.
+
+<img :src="$withBase('/assets/media/xcomponents_func_searchbox.svg')" alt="Search Box"> <br>
+
+::: interact Can't quite capture the concept? Learn more about
+[Search Box](../overview/search-box-overview.md). :::
+
+It can be combined with other X&nbsp;Components, such as [Query Suggestions](query-suggestions.md)
+or [Next Queries](next-queries.md), to modify the query syntax according to your shoppers’
+behaviour. For example, if a shopper selects a query suggestion, the query is instantly updated in
+the input field to the selected suggestion and the search is launched.
+
+:::warning To modify the query syntax using
+[Query Suggestions](../features/query-suggestions-overview.md) or
+[Next Queries](../features/next-queries-overview.md), make sure that your current search service
+supports this type of feature. :::
+
+## Power-up behavior
+
+Every component has a power-up behavior under the hood that is visible to the user but not glaringly
+obvious. For example, in Interface X Components for web, see what happens when:
+
+- _A query with results is **submitted**_:
+
+  - Results are displayed
+  - Related tags are displayed
+  - Next queries are displayed
+  - The query can be displayed in the History Queries or Query Suggestions lists upon configuration
+
+- _A query with results is **cleared** using the clear button_:
+
+  - Any text in the search input is cleared
+  - The results are cleared
+  - The query suggestions are cleared
+  - The next queries are not cleared
+  - The related tags are cleared
+  - The searched query is displayed in history queries
+
+- _Using **instant search and a query is being typed** that has results_:
+
+  - No results are displayed before debounce time
+  - Results are displayed after debounce time
+  - Next queries are displayed after debounce time
+  - Related tags are displayed after debounce time
+
+- _Using **instant search and modifying the initial query**_:
+  - No results related to the second query are displayed before debounce time
+  - Results related to the second query are displayed after debounce time
+  - Displayed results are different from the previous ones
+
+::: warning Deactivating instant search means History Queries are not updated until the search
+button or Enter key is pressed, since the typed query is not submitted until one of these two
+actions is performed. :::
+
+## Tailor the web experience
+
+- Configure the position and place it wherever you prefer.
+- Customize content. Allow shoppers to search or clear their search using text or icons.
+- Determine the number of characters shoppers can enter in the search input.
+- Autofocus the input field when the search&nbsp;box is clicked.
+- Auto-accept the query without the need of using a search button or the Enter key. Determine the
```

Comment:
This is "instant search"  I wouldn't invent new names for an already defined concept.  You can explain the concept if you want, but not giving it a new name.

Reply:
Yes, but here we're describing the action that implies instant search (which is also specifically referred at the end of the sentence). By starting the sentence with _instant search_, we're breaking consistency in the bullet list structure (in other words, parallelism) :)